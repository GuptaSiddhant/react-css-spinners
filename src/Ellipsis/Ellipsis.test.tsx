import React from 'react'
import renderer from 'react-test-renderer'
import { Ellipsis } from './'

it('renders correctly', (): void => {
  const tree = renderer.create(<Ellipsis />).toJSON()
  expect(tree).toMatchSnapshot()
})
