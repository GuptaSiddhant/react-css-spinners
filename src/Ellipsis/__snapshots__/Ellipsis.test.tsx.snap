// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="rcs-ellipsis"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
